client/src/pages/AboutPage.jsx

```diff
@@ -1,12 +1,404 @@
+        icon: LynnBoba,
+      },
+    ],
+  },
+  developers: {
+    categoryTitle: "Software Engineers",
+    members: [
+      {
+        name: "Jenna Xiao",
+        pronouns: "(she/her)",
+        position: "Frontend Software Engineer",
+        linkedIn: "https://www.linkedin.com/in/jenna-x-435a851b3/",
+        website: "",
+        icon: JennaBoba,
+      },
+      {
+        name: "Nancy Tian",
+        pronouns: "(she/her)",
+        position: "Frontend Software Engineer",
+        linkedIn: "http://www.linkedin.com/in/nancy-tian-57934a1b5",
+        website: "",
+        icon: NancyBoba,
+      },
+      {
+        name: "Jessica Chen",
+        pronouns: "(she/they)",
+        position: "Frontend Software Engineer",
+        linkedIn: "https://www.linkedin.com/in/jessica-chen-441222198/",
+        website: "",
+        icon: JessicaBoba,
+      },
+      {
+        name: "Razi Syed",
+        pronouns: "(he/him)",
+        position: "Frontend Software Engineer",
+        linkedIn: "https://www.linkedin.com/in/razisyed-/",
+        website: "",
+        icon: RaziBoba,
+      },
+      {
+        name: "Darren Lee",
+        pronouns: "(he/him)",
+        position: "Backend Software Engineer",
+        linkedIn: "https://www.linkedin.com/in/darrenlee09/",
+        website: "",
+        icon: DarrenBoba,
+      },
+      {
+        name: "Lily Meng",
+        pronouns: "(she/her)",
+        position: "Backend Software Engineer",
+        linkedIn: "https://www.linkedin.com/in/lilyxmeng/",
+        website: "",
+        icon: LilyBoba,
+      },
+      {
+        name: "Anthony Camarillo",
+        pronouns: "(he/him)",
+        position: "Backend Software Engineer",
+        linkedIn: "https://www.linkedin.com/in/anthony-camarillo/",
+        website: "",
+        icon: AnthonyBoba,
+      },
+    ],
+  },
+};
+
+const mentors = [
+  {
+    name: "Brian Lee",
+    pronouns: "(he/him)",
+    linkedInUrl: "https://www.linkedin.com/in/brianscli/",
+    icon: MentorBoba,
+  },
+  {
+    name: "Michael Chan",
+    pronouns: "(he/him)",
+    linkedInUrl: "https://www.linkedin.com/in/michachan",
+    icon: MentorBoba,
+  },
+  {
+    name: "Joshua Wu",
+    pronouns: "(he/him)",
+    linkedInUrl: "https://linkedin.com/in/wujoshua ",
+    websiteUrl: "https://www.joshuawu.me",
+    icon: MentorBoba,
+  },
+  {
+    name: "Emma Li",
+    pronouns: "(she/her)",
+    linkedInUrl: "https://www.linkedin.com/in/emmalili/",
+    icon: MentorBoba,
+  },
+  {
+    name: "Jordan Yep",
+    pronouns: "(he/him)",
+    linkedInUrl: "https://www.linkedin.com/in/jordanyep/ ",
+    icon: MentorBoba,
+  },
+];
 
 const AboutPage = () => {
+  const theme = useTheme();
+
   return (
     <BasePage>
-      <Typography mt={12}>About</Typography>
+      <Stack spacing={5} minWidth="100%" justifyContent="center">
+        <Typography
+          variant="pageTitle"
+          marginTop="6rem"
+          sx={{ width: "100%", borderBottom: "#e1d4c8 solid 1px" }}
```

Comment:
Same question about colors, any theme colors to pick from here over hard coded color values?